v0/destinations/facebook_pixel/transform.js

```diff
@@ -578,6 +579,26 @@ const processEvent = (message, destination) => {
       400
     );
   }
+
+  if (destination.Config.eventsToEvents)
+    destination.Config.eventsToEvents = validateDynamicFormConfig(
+      destination.Config.eventsToEvents,
+      "from",
+      "to"
+    );
+  if (destination.Config.categoryToContent)
+    destination.Config.categoryToContent = validateDynamicFormConfig(
+      destination.Config.categoryToContent,
+      "from",
+      "to"
+    );
+  if (destination.Config.legacyConversionPixelId)
+    destination.Config.legacyConversionPixelId = validateDynamicFormConfig(
+      destination.Config.legacyConversionPixelId,
+      "from",
+      "to"
+    );
+
```

Comment:
Instead of overriding the destination config (which is a bad move), store cleaned up values in separate identifiers.